Simplified/Reader2/Internal/LibraryService.swift

```diff
@@ -191,3 +197,34 @@ final class LibraryService: NSObject, Loggable {
 
 }
 
+extension LibraryService {
+    /*
+     Parse .ncx document after the app creates container and publication.
+     This step is a workaround for current Readium 2 issue with encrypted TOC.
```

Comment:
is this issue tracked somewhere? If so can you add a link to that, so that when it is solved we can remove this code?

Reply:
We can remove this code as soon as R2 adds support for multiple DRMs - currently they open .ncx first (without decrypting it - I think it is not encrypted in case of LCP DRM) and then open other files  in .epub using DRM. I asked Mickaël about that part, he says it won't be a problem once multiple DRM is supported

Reply:
gotcha, I just created https://jira.nypl.org/browse/SIMPLY-2840 for this cleanup. We have a convention of adding a  `TODO: SIMPLY-2840` note in a comment wherever that particular ticket requires intervention, so here would be a good place. If you can make sure to add that note in other places where cleanup is needed after R2 completes multiple DRM support, that would be fantastic.

Reply:
done! added TODO to all files that should be removed

Reply:
thanks!